pyvista/core/filters.py

```diff
@@ -1091,10 +1091,10 @@ def warp_by_scalar(dataset, scalars=None, factor=1.0, normal=None,
         Parameters
         ----------
         scalars : str, optional
-            Name of scalars to warb by. Defaults to currently active scalars.
+            Name of scalars to warp by. Defaults to currently active scalars.
 
         factor : float, optional
-            A scalaing factor to increase the scaling effect. Alias
+            A scaling factor to increase the scaling effect. Alias
```

Comment:
I've also spotted some typos in adjacent docstrings, so here's a suggestion for fixing them.

Reply:
thanks!